apps/myaccount/src/helpers/history.ts

```diff
@@ -16,6 +16,8 @@
  * under the License.
  */
 
-import { createBrowserHistory } from "history";
+import { History, createBrowserHistory } from "history";
 
-export const history = createBrowserHistory({ basename: window["AppUtils"]?.getConfig()?.appBaseNameForHistoryAPI });
+
+// eslint-disable-next-line max-len
+export const history: History =  createBrowserHistory({ basename: window["AppUtils"]?.getConfig()?.appBaseNameForHistoryAPI });
```

Comment:
Do not disable eslint rules.

Suggestion to resolve the max-len issue
```suggestion
export const history: History =  createBrowserHistory({
    basename: window["AppUtils"]?.getConfig()?.appBaseNameForHistoryAPI
});
```

Reply:
Ack